src/EnergyPlus/SizingManager.cc

```diff
@@ -2809,7 +2809,7 @@ void GetSizingParams(EnergyPlusData &state)
     } else {
         ShowFatalError(state, cCurrentModuleObject + ": More than 1 occurrence of this object; only 1 allowed");
     }
-
+    if (state.dataGlobal->OverrideTimestep) state.dataSize->NumTimeStepsInAvg = state.dataGlobal->NumOfTimeStepInHour;
```

Comment:
@yujiex Are you planning to finish this for v22.1?
When OverrideTimestep is true, this will force the averaging window to be one hour, regardless of what the user input averaging window is. That's probably ok and better than current behavior. But do we want this be sensitive to the original user inputs for averaging window and timestep?
`state.dataSize->NumTimeStepsInAvg = round(state.dataGlobal->NumOfTimeStepInHour * state.dataSize->NumTimeStepsInAvg / state.dataGlobal->UserInputNumOfTimeStepInHour;)`

Also, this should probably throw a warning here?

Reply:
Right, I'll add a warning message.

It's probably better to get the user input to take effect. I'll try to fix it.

Reply:
Sorry, I misunderstood the PR. So the intended behavior would be:
If the fast simulation mode is selected, the OverrideTimestep flag will be set to True, which will force the averaging window to be one hour, regardless of what the user input averaging window is.